### PR TITLE
[Quantum RPG] Finish frontier zone

### PR DESCRIPTION
**Pull request by dstrain115**
- Add some more rooms and items
- Add bridge to next zone.
- With this PR, we should be good to go for a simple demo!

### CODE REVIEW

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py [hunk 1]
```diff
@@ -84,6 +100,69 @@
     ]
 )
 
+
+def _hole(state: GameState, world):
+    raise UntimelyDeathException(
+        "You feel your entire existence dissolve into errors and static\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:106]: DEL static
You repeat static on the next line.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py [hunk 2]
```diff
@@ -84,6 +100,69 @@
...
+    )
+
+
+HOLE = Item(
+    keyword_actions=[
+        (
+            EXAMINE,
+            ["darkness", "vista", "hole", "static", "piece", "area"],
+            (
+                "It looks like this used to be a path to a room, but is now just an\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:117]: REVERSE ORDER is, now

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py [hunk 3]
```diff
@@ -84,6 +100,69 @@
...
+                "empty void, filled with pixelated static.  This must be a form of the\n"
+                "errors that you have heard about.  You think it would be very unwise to\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:119]: DEL heard

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py [hunk 4]
```diff
@@ -84,6 +100,69 @@
...
+        return "The engineer uses nearby logs to repair the bridge and provide a safe passage."
+
+
+BRIDGE = Item(
+    keyword_actions=[
+        (
+            EXAMINE,
+            ["bridge"],
+            (
+                "It looks like as if this bridge has fallen into disrepair.\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:151]: DEL like OR as if

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py [hunk 5]
```diff
@@ -84,6 +100,69 @@
...
+                "Parts have decayed and have fallen into the river. However,\n"
+                "there are plenty of logs around, and the supports are intact,\n"
+                "so it should be possible to repair with someone with the right skills."
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:154]: REPLACE with
WITH it, for

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py [hunk 6]
```diff
@@ -199,7 +278,54 @@
...
+            "slope downwards towards a river that winds its way across the frontier lands."
+        ),
+        encounters=[blue_foam(2, 0.2), red_foam(2, 0.2)],
+        exits={Direction.SOUTH: "classical8", Direction.NORTH: "classical10"},
+    ),
+    Location(
+        label="classical10",
+        title="Sloping Path",
+        description=(
+            "The path slopes downward towards a rushingriver to the north.\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:298]: INSERT SPACE rushingriver

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py [hunk 7]
```diff
@@ -199,7 +278,54 @@
...
+        ),
+        encounters=[red_foam(2, 0.3), green_foam(3, 0.2)],
+        exits={Direction.SOUTH: "classical9", Direction.NORTH: "classical11"},
+    ),
+    Location(
+        label="classical11",
+        title="At the Bank of a Rushing River",
+        description=(
+            "You have reached the southern shore of the river which marks the\n"
+            "the border of the quantum realm.  Beyond, you can see Hadamard hills\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:311]: INSERT the
BEFORE Hadamard
CAPITALIZE hills

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py [hunk 8]
```diff
@@ -199,7 +278,54 @@
...
+        exits={Direction.SOUTH: "classical10", Direction.EAST: "classical12"},
+    ),
+    Location(
+        label="classical12",
+        title="At a Broken Bridge",
+        description=(
+            "Here, at the southern shore of the river is a rickety bridge\n"
+            "that leads to the northern side of the river. Pieces of the bridge\n"
+            "have collapsed and fallen apart, and there seems to be no way to\n"
+            "cross safely in the condition the bridge is in now."
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:325]: REPLACE safely in
WITH safely, given

**Review by cognigami — CHANGES_REQUESTED** [review]

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py (hunk 1 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:106]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py (hunk 2 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:117]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py (hunk 3 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:119]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py (hunk 4 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:151]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py (hunk 5 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:154]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py (hunk 6 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:298]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py (hunk 7 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:311]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py (hunk 8 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py:325]: Done.

**Review by cognigami — APPROVED** [review]

**Pull request merged**